core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/impl/aggregation/aggregators/AggregatorOptionSwitch.kt

```diff
@@ -31,7 +30,7 @@ internal class AggregatorOptionSwitch1<in Param1, in Value : Any, out Return : A
          *   MyAggregator.Factory(param1)
          * }
          */
-        fun <Param1, Value : Any, Return : Any?> Factory(
+        internal fun <Param1, Value : Any, Return : Any?> Factory(
```

Comment:
if we make everything public, the factory can also be public imo

Reply:
I will check this place twice, it was a tricky thing to make public not all the stuff